layers/+lang/csharp/config.el

```diff
@@ -29,3 +29,4 @@
   "The backend to use for IDE features.
 Possible values are `omnisharp' and `lsp'.
 If `nil' then no backend is enabled.")
+(put 'csharp-backend 'safe-local-variable #'symbop)
```

Comment:
typo `symbolp`

Reply:
I'm going to replace the declaration by spacemacs|defc and the safe-local-variables property would be set correctly.